azure-pipelines.yml

```diff
@@ -7,21 +7,22 @@ pool:
   vmImage: 'Ubuntu 16.04'
 
 steps:
-- script: |
-    sudo npm install -g typescript
-  displayName: install typescript
-
-- script: npm run build
-  displayName: run npm based build
+- script: node -v; npm -v; npm install
+  displayName: npm install
 
 # - script: |
 #    cd tests/rotate
 #    docker run -v "$PWD":/code --rm --workdir /code node node  /code/node.js
 #  displayName: test wasm with nodejs
 
-- script: npm run test 
+- script: npm test
   displayName: run tests
   
+
+- script: npm run test-browser
```

Comment:
working ! : https://dev.azure.com/oneeyedelf1/wasm-imagemagick/_build/results?buildId=78&view=logs

maybe we want that npm test to run both things so we have a single `npm test` command here ?